Buscorepuestos/buscorepuestos-web
# Feature/br 358 slider

## What type of PR is this? (check all applicable)


## Description
The responsive view and the view of how many cards are visible on the screen were developed.


## Related Tickets & Documents
[](https://linear.app/buscorepuestos/issue/BR-358/slider)

## QA Instructions, Screenshots, Recordings

Verfify swipers components, cardPrice and CardInfo

Also make sure you check this browser compatibility version for desktop and mobile:

## Checklist

DO NOT send the PR without completing the checklist above:


## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

      have not been included_

## Are there any post deployment tasks we need to perform?
